pom.xml

```diff
@@ -1517,7 +1518,7 @@
         <maven-surefire-plugin.version>3.0.0-M6</maven-surefire-plugin.version>
         <openrewrite.version>7.22.0</openrewrite.version>
         <swagger-parser-groupid.version>io.swagger.parser.v3</swagger-parser-groupid.version>
-        <swagger-parser.version>2.1.1</swagger-parser.version>
+        <swagger-parser.version>2.1.6</swagger-parser.version>
```

Comment:
@OpenAPITools/generator-core-team FYI. swagger parser and other dependencies version update.

Reply:
It solve multiple cve. It's this part that make the contract validation more accurate.